Libraries/acdriver/CMakeLists.txt

```diff
@@ -14,11 +14,17 @@ add_library(acdriver SHARED
             Sources/Result.cpp
             Sources/VersionAction.cpp
             Sources/CompileAction.cpp
+            Sources/CompileActionImageSet.cpp
+            Sources/CompileOutput.cpp
```

Comment:
I don't see why this needs to be split up. Are there any existing tests in xcbuild for CLIs?